modules/virt-adding-kernel-arguments-enable-iommu.adoc

```diff
@@ -0,0 +1,58 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/advanced_vm_management/configuring-pci-passthrough.adoc
+
+[id="virt-adding-kernel-arguments-to-enable-iommu-for-the-bios_{context}"]
+= Adding kernel arguments to enable IOMMU for the BIOS
+
+To enable an IOMMU extension for the BIOS (Basic Input/Output System) associated with a PCI device, create the `MachineConfig` object file and add the kernel arguments for the device.
+
+.Prerequisites
+* Administrative privilege to a working {product-title} cluster.
+* Intel or AMD CPU hardware.
+* Enable the hardware virtualization extensions for Intel Virtualization Technology (Intel VT-X). AMD-V extensions are typically enabled by default.
++
```

Comment:
and add 
```
Enable the Virtualization Technology for Directed I/O extensions VT-d or AMD IOMMU in the BIOS.
```

Reply:
I have taken your text and slightly adjusted the text to read: 
Enable Intel Virtualization Technology for Directed I/O extensions or AMD IOMMU in the BIOS (Basic Input/Output System).